frontend/test/lit/test_peephole_optimizations.py

```diff
@@ -150,20 +121,22 @@ def f(x):
     def test_pipeline_lowering_keep_original_workflow(x):
         return f(x), f_pipeline(x)
 
-    # CHECK: transform_named_sequence
     # CHECK: call_jaxpr=
-    # CHECK-NOT: _:AbstractTransformMod() = apply_registered_pass[
+    # CHECK: transform_named_sequence2
     # CHECK: call_jaxpr=
+    # CHECK: transform_named_sequence2
     # CHECK: _:AbstractTransformMod() = apply_registered_pass[
     # CHECK:   options=func-name=f_transformed0
     # CHECK:   pass_name=remove-chained-self-inverse
-    # CHECK: ]
     # CHECK: _:AbstractTransformMod() = apply_registered_pass[
     # CHECK:   options=func-name=f_transformed0
     # CHECK:   pass_name=merge-rotations
     # CHECK: ]
     print_jaxpr(test_pipeline_lowering_keep_original_workflow, 1.2)
 
+    # COM: This is the one that is unchanged
```

Comment:
Just curious: we did not have to change how that `apply_pass` primitives were binded in `frontend/passes.py`?

Reply:
No, but I do want to clean it up a bit in a following PR.